### ISSUE TITLE
Update sign-in alerts to match content in VADS

### ISSUE THREAD

**Issue by sterkenburgsara** [department-of-veterans-affairs/va.gov-team#96243]
## Description
Content for all of our access alerts have gone through new rounds of approvals and have been published for sitewide usage across any applications that require LOA3 users. 

There are new alert component designs published in VADS [here](https://design.va.gov/components/alert/alert-sign-in/), however, they are NOT YET available as coded content components that we can grab and directly put into alerts. The design system team is planning to work on that during an upcoming sprint slated for Nov. 4-18, but in the meantime we would like to at least update our alert content to match the finalized versions. 

A follow-on ticket will come later (once those content components are ready) to pivot from our custom components to using the VADS ones. 

### User story

As a Cartography team member, I want to update the content in our access / sign-in alerts to match the finalized versions now available in VADS, so that all users who encounter these alerts across VA.gov will see consistent versions and language everywhere. 

### Notes
- ~~The "register with My HealtheVet" alert specifically is much improved / way shorter content. Possibly prioritize fixing this alert first~~ Remove the updating of "register with My HealtheVet" alert from this ticket as we'll want to have a UX mock before starting the engineering effort
- Our current implementation uses action links, not buttons - these buttons are not yet built, so we need to leave the action links & only edit the body text content 
- Future work to replace alert component with newer version: https://github.com/department-of-veterans-affairs/va.gov-team/issues/90884

## Possible tasks:
- [ ] UX to mock up the alerts with new VADS body text content, but still utilizing action links (mock w/o the verification buttons)
- [ ] Engineering to update the body text content to match what's in the Figma handed off from UX

## Acceptance criteria
- [ ] Alerts have the most updated / standardized text body language
- [ ] Alerts still have action links (Not buttons - we'll update those when the VADS components are ready to use) 
- [ ] UX review
- [x] QA review
- [ ] A11y review

**Comment by nichia** [comment]
Per today’s conversation with Sara, the scope of this ticket is changed to update the heading and body text content for two sign-in alerts ([Verify with ID.me](https://design.va.gov/components/alert/alert-sign-in/#verify-with-idme) and [Verify with Login.gov](https://design.va.gov/components/alert/alert-sign-in/#verify-with-logingov)) on the MHV landing page to match the VSDS sign-in alerts.
The third sign-in alerts ([Sign in with a different account](https://design.va.gov/components/alert/alert-sign-in/#sign-in-with-a-different-account)) will remain unchanged, pending UX to mock up with the new VADS body text content.

**Comment by allanto-ah** [comment]
Environment: Staging

Verified that the heading and body text have been updated to match the content in the VADS. 

### id.me
![Image](https://github.com/user-attachments/assets/9339621b-9bbe-420a-935c-c57c6de110f4)

### login.gov
![Image](https://github.com/user-attachments/assets/0f15b14e-3b8e-419d-acdf-5ec64fa50a43)
